Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/BraveSkusScriptHandler.swift

```diff
@@ -16,17 +16,28 @@ class BraveSkusScriptHandler: TabContentScript {
   typealias ReplyHandler = (Any?, String?) -> Void
   
   private weak var tab: Tab?
-  
   private let sku: SkusSkusService?
   
   required init(tab: Tab) {
     self.tab = tab
     self.sku = Skus.SkusServiceFactory.get(privateMode: tab.isPrivate)
   }
-  
-  static func name() -> String { "BraveSkusHelper" }
-  
-  func scriptMessageHandlerName() -> String? { "braveSkusHelper" }
+    
+  static let scriptName = "BraveSkusScript"
+  static let scriptId = UUID().uuidString
+  static let messageHandlerName = "\(scriptName)_\(messageUUID)"
+  static let scriptSandbox: WKContentWorld = .page
+  static let userScript: WKUserScript? = {
+    guard var script = loadUserScript(named: scriptName) else {
+      return nil
+    }
+    return WKUserScript.create(source: secureScript(handlerName: messageHandlerName,
+                                                    securityToken: scriptId,
+                                                    script: script),
+                               injectionTime: .atDocumentStart,
+                               forMainFrameOnly: false,
```

Comment:
I think we want it for main frame only. See
https://github.com/brave/brave-ios/blob/development/Client/Frontend/Browser/User%20Scripts/ScriptFactory.swift#L128

Would be good to double check all scripts and see if we did not changed their main frame requirement by mistake.

Reply:
Fixed